core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXV1Controller.java

```diff
@@ -587,6 +588,12 @@ public DistroXSyncCmV1Response syncComponentVersionsFromCmByCrn(@ResourceCrn Str
         return launchSyncComponentVersionsFromCm(NameOrCrn.ofCrn(crn));
     }
 
+    @Override
+    @CheckPermissionByResourceName(action = AuthorizationResourceAction.POWERUSER_ONLY)
+    public void changeImageCatalog(@ResourceName @NotEmpty String name, @Valid @NotNull ChangeImageCatalogV4Request changeImageCatalogRequest) {
+        // FIXME: Implement this!
```

Comment:
API first approach so we can work in parallel on the CLI.
It is going to be implemented as a part of CB-14308

Reply:
I think this comment would be better in the code, not just the PR